externals/kyuubi-jdbc-engine/src/main/scala/org/apache/kyuubi/engine/jdbc/connection/ConnectionProvider.scala

```diff
@@ -57,10 +57,9 @@ abstract class AbstractConnectionProvider extends Logging {
       case None =>
         // TODO
         if (filteredProviders.size != 1) {
-          throw new IllegalArgumentException(
```

Comment:
filter with the full class name here

Reply:
providers filtered by `driverClass` , and `kyuubi.engine.jdbc.connection.provider` is not set, how to filter with the full class name here?
```
providers.filter(_.canHandle(getProviderClass(kyuubiConf)))
```

Reply:
@pan3793 cc

Reply:
I see, method name `getProviderClass` is confusing, let's rename it to `getDriverClass`, other LGTM

Reply:
OK